ferveo/src/bindings_python.rs

```diff
@@ -268,21 +268,28 @@ pub fn decrypt_with_shared_secret(
 }
 
 #[pyclass(module = "ferveo")]
-struct FerveoVariant {}
+pub struct FerveoVariant {}
 
 #[pymethods]
 impl FerveoVariant {
-    #[staticmethod]
+    #[classattr]
     fn precomputed() -> &'static str {
```

Comment:
Remind me: why do we need to deal with `str` values (to/from) at all as opposed to continuing to use the `api::FerveoVariant::<Simple|Precomputed>` enum directly?

Reply:
It's not possible to use `api::FerveoVariant` enum directly here since it's shared by WASM and Python bindings. That being said, I'd prefer to expose a Python `Enum` here instead of `str`. Initially, I didn't find a way to do that without rewriting `api::FerveoVariant` from scratch in `bindings_python.rs`, but I will look into it again today.